### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@DeepDiver1975 requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/owncloud/core/

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
:+1:

File: apps/dav/lib/systemtag/systemtagsbyidcollection.php [hunk 1]
```diff
@@ -46,14 +46,23 @@ public function __construct($tagManager) {
 		$this->tagManager = $tagManager;
 	}
 
+	/**
+	 * @param string $name
+	 */
 	function createFile($name, $data = null) {
```
Comment (nickvergessen) [inline apps/dav/lib/systemtag/systemtagsbyidcollection.php]: data missing :see_no_evil:

File: apps/dav/lib/systemtag/systemtagsobjecttypecollection.php [hunk 2]
```diff
@@ -61,14 +61,23 @@ public function __construct($objectType, $tagManager, $tagMapper) {
 		$this->objectType = $objectType;
 	}
 
+	/**
+	 * @param string $name
+	 */
 	function createFile($name, $data = null) {
```
Comment (nickvergessen) [inline apps/dav/lib/systemtag/systemtagsobjecttypecollection.php]: same here

**Comment by DeepDiver1975** [comment]
@nickvergessen mind a second look? THX

**Comment by nickvergessen** [comment]
:+1:

**Pull request merged**